app/customers.py

```diff
@@ -0,0 +1,55 @@
+from math import sqrt
+import datetime
+
+from app.car import Car
+from app.shops import Shops
+
+
+class Customers:
+
+    def __init__(self, name: str,
+                 product_cart: dict,
+                 customer_location: list,
```

Comment:
List of what? Add more descriptive typehint, please. For example, if It's a list of integers: list[int] and so on